frontend/src/app/shared/enums/metric.enum.ts

```diff
@@ -0,0 +1,19 @@
+import {Unit} from "./unit.enum";
+
+export class Metrics {
```

Comment:
please use a json object (e.g.: "export const metrics = { SPEED_INDEX: Metric = new Metric (.... ), ... }) instead of a class with methods

Reply:
done.